mesa/time.py

```diff
@@ -39,12 +39,24 @@
 
 
 class BaseScheduler:
-    """Simplest scheduler; activates agents one at a time, in the order
-    they were added.
-
-    Assumes that each agent added has a *step* method which takes no arguments.
-
-    (This is explicitly meant to replicate the scheduler in MASON).
+    """
+    A simple scheduler that activates agents one at a time, in the order they were added.
+
+    This scheduler is designed to replicate the behavior of the scheduler in MASON, a multi-agent simulation toolkit.
+    It assumes that each agent added has a `step` method which takes no arguments and executes the agent's actions.
+
+    Attributes:
+        model (Model): The model instance associated with the scheduler.
+        steps (int): The number of steps the scheduler has taken.
+        time (TimeT): The current time in the simulation. Can be an integer or a float.
+        _agents (dict[int, Agent]): A private dictionary mapping agent IDs to agent instances.
```

Comment:
This shouldn't be documented. The private implementation detail may change, while the API (`BaseScheduler.agents`) doesn't.

Reply:
`model.schedule.steps` (or `self.schedule.steps`) is frequently used to get the current timestep. I think this should be documented somewhere. Or is there another best practice to get the current timestep?

Reply:
Just `_agents`. Everything else is fine.

Reply:
I removed `_agents` from the docs.